src/features/offer/components/shareMessagingOffer/InstalledMessagingApps.tsx

```diff
@@ -38,12 +39,13 @@ export const InstalledMessagingApps = ({ offerId }: { offerId: number }) => {
           isNative,
           ...options
         } = mapNetworkToSocial[network]
-        // Message has to be concatenated with url if url option is not supported
-        const message = supportsURL ? shareMessage : shareMessage + '\n' + shareUrl
+        // Message has to be concatenated with url if url option is not supported, and in web
+        const message = supportsURL && !isWeb ? shareMessage : shareMessage + '\n' + shareUrl
```

Comment:
Je comprends pas le commentaire par rapport à ce que tu as fait ensuite ? Tu dis "and in web" mais ta condition exclue le web 🧐

Reply:
Le message doit être concaténé en web, dans le ternaire j'exclue le web parce que la concaténation se trouve dans le "else" du ternaire

Reply:
Ok, j'ai l'impression qu'on perd un peu en clarté en le mettant ici, mais comme tu veux !